src/main/java/eu/adlogix/appnexus/oas/client/utils/ValidatorUtils.java

```diff
@@ -0,0 +1,26 @@
+package eu.adlogix.appnexus.oas.client.utils;
+
+import java.util.Collection;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+public class ValidatorUtils {
```

Comment:
Because Google methods didn't do exactly what i wanted. Ex: checkNotNull will throw a NullPointer and not a runtime. Since the implementation is quite simple, I wrote this class using core Java